src/lib/edge-functions/registry.mjs

```diff
@@ -207,6 +195,14 @@ export class EdgeFunctionsRegistry {
       )
 
       this.#processGraph(graph)
+
+      if (npmSpecifiersWithExtraneousFiles.length !== 0) {
+        const modules = npmSpecifiersWithExtraneousFiles.map((name) => chalk.yellow(name)).join(', ')
+
+        log(
+          `${NETLIFYDEVWARN} The following npm modules, which are directly or indirectly imported by an edge function, may not be supported: ${modules}. For more information, visit https://ntl.fyi/edge-npm.`,
```

Comment:
this isn't pointing anywhere yet. i'm assuming you want to point this at the docs, or at the blog post?

Reply:
The blog post won't cover this directly. I think we'd point it either at the docs or at the forums post.